docs/tutorial.md

```diff
@@ -349,6 +349,137 @@ If the deployment is complete, `plan.status` in the returned JSON
 should equal "COMPLETE".  If it is, you should see two `meta-data`
 tasks and three `data` tasks running under the service.
 
+# Plan management
+As noted earlier, stateful services often have particular deployment 
```

Comment:
Users may better understand this section if you start it with the motivating feature: "you may want to manually step through an install or upgrade for (insert reason here), the SDK creates *plans* on your behalf which allow you to do this"